cirq-google/cirq_google/engine/util_test.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 The Cirq Developers
```

Comment:
this file is called `util_test.py` but doesn't test anything from `util.py`

Reply:
These are utilities that can be _used_ in tests. Happy to rename the file if you have a suggestion for something better.

Reply:
test_utils?

Reply:
Done.